### PR TITLE
Re-organization of tutorials

### PR DESCRIPTION
**Pull request by mdenolle**
rearanged the TOC

### CODE REVIEW

**Review by carlosgjs — CHANGES_REQUESTED** [review]
Looks like some test  .h5 got accidentally added and need to be removed. Once removed I suggest adding `tutorials/asdf_data` to the [.gitignore](https://github.com/mdenolle/NoisePy/blob/01ba3de51631dccdd53a35691fc287852313cb4f/.gitignore#L176)

Also, it's a best practice to clear the output of the notebooks before comitting them.

File: tutorials/noisepy_scoped_download_s3_store.ipynb [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
```
Comment (carlosgjs) [inline tutorials/noisepy_scoped_download_s3_store.ipynb:69]: Please clear all the cell outputs. Jupyter Notebook: `Edit -> Clear All Outputs`, or in VS Code:

image

**Review by carlosgjs — CHANGES_REQUESTED** [review]
See one more cleanup comment inline.

Also, can you give the PR a more descriptive name? When merging via "Squash and Merge" (preferred method), the PR title becomes the commit message in the history so its useful if it reflects the purpose of the PR. A good practice is to use the [Conventional Commit](https://gist.github.com/qoomon/5dfcdf8eec66a051ecd85625518cfd13) format.

File: tutorials/noisepy_scoped_download_s3_store.ipynb (hunk 1 above)
Comment (mdenolle) [reply tutorials/noisepy_scoped_download_s3_store.ipynb:69]: cleared up

**Review by niyiyu — APPROVED** [review]

**Review by carlosgjs — APPROVED** [review]

**Pull request merged**